programs/perpetuals/tests/anchor/cryptography.ts

```diff
@@ -0,0 +1,6 @@
+import { Keypair } from "@solana/web3.js";
```

Comment:
why do we need a separate file just for a single line? keep it simple

Reply:
done, removed and replaced caller with the one liner `nacl.sign.detached`